src/components/ui/store/ProductList.tsx

```diff
@@ -127,51 +123,43 @@ export const ProductRepeater = ProductListPrimitive.ProductRepeater;
  * ```
  */
 export const LoadMoreTrigger = React.forwardRef<
```

Comment:
I know that its not related to the PR but the name of the component shouldn't be  ProductLoadMoreTrigger